atlasdb-api/src/main/java/com/palantir/atlasdb/transaction/api/Transaction.java

```diff
@@ -65,7 +65,7 @@
      * @return a mapping of rows to the columns matching the provided column selection
      */
     @Idempotent
-    SortedMap<byte[], RowResult<byte[]>> getRows(
+    NavigableMap<byte[], RowResult<byte[]>> getRows(
```

Comment:
isn't this a binary backcompat break?

Reply:
Yes, just hit this. It seems that anyone relying on internal libraries (like atlas-event-queue) will need to upgrade as they get errors like this:

```
java.lang.NoSuchMethodError: 'java.util.SortedMap com.palantir.atlasdb.transaction.api.Transaction.getRows(com.palantir.atlasdb.keyvalue.api.TableReference, java.lang.Iterable, com.palantir.atlasdb.keyvalue.api.ColumnSelection)'
	at com.palantir.atlas.sequential.schema.generated.OffsetsTable.getRows(OffsetsTable.java:613)
	at com.palantir.atlas.sequential.store.OffsetsStore.getOffsets(OffsetsStore.java:79)
	at com.palantir.atlas.sequential.store.OffsetsStore.getLatestOffsets(OffsetsStore.java:40)
	at com.palantir.atlas.sequential.OrderedTableReader.getLatestOffsets(OrderedTableReader.java:95)
	at com.palantir.atlas.sequential.ObjectMappingReader.getLatestOffsets(ObjectMappingReader.java:46)
	at com.palantir.atlas.sequential.api.SequentialTableReader.getLatestOffset(SequentialTableReader.java:81)
	at com.palantir.atlas.eventdriven.AtlasEventReader.lambda$getLatestOffset$1(AtlasEventReader.java:34)
```